packages/esm-patient-notes-app/src/notes/visit-notes-form.workspace.tsx

```diff
@@ -289,6 +276,9 @@ const VisitNotesForm: React.FC<DefaultPatientWorkspaceProps> = ({
     const close = showModal('capture-photo-modal', {
       saveFile: (file: UploadedFile) => {
         if (file) {
+          if (file.capturedFromWebcam && !file.fileName.includes('.')) {
+            file.fileName = `${file.fileName}.png`;
+          }
           setValue('images', currentImages ? [...currentImages, file] : [file]);
         }
```

Comment:
```suggestion
        if (file.capturedFromWebcam && !file.fileName.includes('.')) {
          file.fileName = `${file.fileName}.png`;
        }

        setValue('images', currentImages ? [...currentImages, file] : [file]);
```

Maybe we can simplify this like so.